cluster/sync.sh

```diff
@@ -2,80 +2,106 @@
 
 set -ex
 
+kubectl=./cluster/kubectl.sh
+
 function getDesiredNumberScheduled {
-    echo $(./cluster/kubectl.sh get daemonset -n nmstate $1 -o=jsonpath='{.status.desiredNumberScheduled}')
+    $kubectl get daemonset -n nmstate $1 -o=jsonpath='{.status.desiredNumberScheduled}'
 }
 
 function getNumberAvailable {
-    numberAvailable=$(./cluster/kubectl.sh get daemonset -n nmstate $1 -o=jsonpath='{.status.numberAvailable}')
+    numberAvailable=$($kubectl get daemonset -n nmstate $1 -o=jsonpath='{.status.numberAvailable}')
     echo ${numberAvailable:-0}
 }
 
+function eventually {
+    timeout=15
```

Comment:
These changes are unrelated to the change in generating of manifests, right? Please open a dedicated PR for that.

Reply:
We need them to make CI happy, and they change same script, we don't need to backport this since we don't have webhooks at release-0.15